backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/hostdeploy/InstallVdsInternalCommand.java

```diff
@@ -277,7 +277,9 @@ private void runAnsibleHostDeployPlaybook() {
             }
             Cluster hostCluster = clusterDao.get(getClusterId());
             boolean isGlusterServiceSupported = hostCluster.supportsGlusterService();
-            String tunedProfile = isGlusterServiceSupported ? hostCluster.getGlusterTunedProfile() : null;
+            boolean isVirtEnabled = hostCluster.supportsVirtService();
+            String tunedProfile = isVirtEnabled && !isGlusterServiceSupported ? "virtual-host"
+                    : isGlusterServiceSupported ? hostCluster.getGlusterTunedProfile() : null;
```

Comment:
Does this fix the issue that gluster services are enabled for the cluster, but gluster tuned profile is set to NULL due to the UI bug? Wouldn't below condition be more clear?

```
String tunedProfile = "virtual-host";
if (hostCluster.supportsGlusterService() && StringUtils.isNotBlank(hostCluster.getGlusterTunedProfile())) {
    // different tuned profile is specified for gluster cluster
    tunedProfile = hostCluster.getGlusterTunedProfile();
}
```

Reply:
yes, done